src/Sylius/Component/Core/Promotion/Checker/Rule/ContainsProductRuleChecker.php

```diff
@@ -34,14 +37,13 @@ public function isEligible(PromotionSubjectInterface $subject, array $configurat
 
```

Comment:
Not visible, but above this line we should use `Assert::instanceOf` instead of `if`